internal/cmd/keys.go

```diff
@@ -70,21 +73,24 @@ $ infra keys add example-key machine-a --ttl=12h
 
 			resp, err := client.CreateAccessKey(&api.CreateAccessKeyRequest{
 				IdentityID:        machine.ID,
-				Name:              keyName,
+				Name:              options.Name,
 				TTL:               api.Duration(options.TTL),
 				ExtensionDeadline: api.Duration(options.ExtensionDeadline),
 			})
 			if err != nil {
 				return err
 			}
 
-			cli.Output("key: %s", resp.AccessKey)
+			fmt.Fprintf(os.Stderr, "Created access key for %q\n", machineName)
+			fmt.Printf("Name: %s\n", resp.Name)
+			fmt.Printf("Key: %s\n", resp.AccessKey)
```

Comment:
So that we can capture these in tests:

```suggestion
			fmt.Fprintf(cli.Stderr, "Created access key for %q\n", machineName)
			cli.Output("Name: %s", resp.Name)
			cli.Output("Key: %s", resp.AccessKey)
```